errors.go

```diff
@@ -57,6 +60,9 @@ func SetSpanTags(span opentracing.Span, err error, client bool) {
 	code := codes.Unknown
 	if s, ok := status.FromError(err); ok {
 		code = s.Code()
+	} else if errors.Is(err, context.Canceled) {
```

Comment:
We also need to check for `code == codes.Canceled` here - the gRPC client libraries translate client-side context cancellations into gRPC errors with code `Canceled` using either [`FromContextError`](https://github.com/grpc/grpc-go/blob/c0aa20a8ac825f86edd59b2cab842de6da77a841/status/status.go#L151) or [`toRPCErr`](https://github.com/grpc/grpc-go/blob/c0aa20a8ac825f86edd59b2cab842de6da77a841/rpc_util.go#L874) depending on when the error is observed by the client.

`TestStreamingContextCancellationOpenTracing` doesn't cover this case because it doesn't test cancelling the context while a client is in a `Recv()` or `Send()` call - it cancels the context while the stream is open, but not while the client is in `Recv()` or `Send()`.

Reply:
Thanks; I have modified the code to check `codes.Canceled`

Reply:
Thanks. Could you please add a test to cover this case as well?